src/Attributes/OpenApi.php

```diff
@@ -6,17 +6,19 @@
 
 namespace OpenApi\Attributes;
 
+use OpenApi\Annotations\PathItem;
```

Comment:
Is this necessary? There is already a `PathItem` class in the same namespace?

Reply:
wrong use, thx